purescript.cabal

```diff
@@ -391,6 +396,11 @@ executable purs
                    Command.REPL
     ghc-options: -Wall -O2 -fno-warn-unused-do-bind -threaded -rtsopts "-with-rtsopts=-N"
 
+    if flag(release)
+      cpp-options: -DRELEASE
+    else
+      build-depends: gitrev >= 1.2.0 && <1.3
```

Comment:
Presumably we don't need to update the license because we don't ship with this library. Does `license-generator` handle that properly?